test/ng/parseSpec.js

```diff
@@ -3232,6 +3232,18 @@ describe('parser', function() {
           expect(fn()).toEqual(undefined);
         }));
 
+        it('should invoke a stateless filter once when the parsed expression has an interceptor',
+           inject(function($parse, $rootScope) {
+          var countFilter = jasmine.createSpy();
+          var interceptor = jasmine.createSpy();
+          countFilter.and.callThrough();
```

Comment:
But, how is `.and.callThrough()` changing anything. You don't have an actual function to call through anyway.